pkgs/development/python-modules/mdformat-wikilink/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  mdformat,
+  poetry-core,
```

Comment:
@natsukium Thanks for review! Committed all suggestions. Should `pytest-cov` still be here?